### PR TITLE
Add iOS platform

### PR DESCRIPTION
**Pull request by stephanepechard**
Tests are all green for iOS.

### CODE REVIEW

**Comment by stephanepechard** [comment]
I updated the version and the README file.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 Cron realization for Kotlin Multiplatform
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Scogun_kcron-common&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Scogun_kcron-common) ![GitHub](https://img.shields.io/github/license/Scogun/kcron-common?color=blue) ![Publish workflow](https://github.com/Scogun/kcron-common/actions/workflows/publish.yml/badge.svg) [![Maven Central with version prefix filter](https://img.shields.io/maven-central/v/com.ucasoft.kcron/kcron-common/0.5.3?color=blue)](https://search.maven.org/artifact/com.ucasoft.kcron/kcron-common/0.5.3/jar)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Scogun_kcron-common&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Scogun_kcron-common) ![GitHub](https://img.shields.io/github/license/Scogun/kcron-common?color=blue) ![Publish workflow](https://github.com/Scogun/kcron-common/actions/workflows/publish.yml/badge.svg) [![Maven Central with version prefix filter](https://img.shields.io/maven-central/v/com.ucasoft.kcron/kcron-common/0.5.4?color=blue)](https://search.maven.org/artifact/com.ucasoft.kcron/kcron-common/0.5.3/jar)
```
Comment (Scogun) [inline README.md:4]:
```suggestion
[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Scogun_kcron-common&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Scogun_kcron-common) ![GitHub](https://img.shields.io/github/license/Scogun/kcron-common?color=blue) ![Publish workflow](https://github.com/Scogun/kcron-common/actions/workflows/publish.yml/badge.svg) [![Maven Central with version prefix filter](https://img.shields.io/maven-central/v/com.ucasoft.kcron/kcron-common/0.5.4?color=blue)](https://search.maven.org/artifact/com.ucasoft.kcron/kcron-common/0.5.4/jar)
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
 ### Features
-* Kotlin Multiplatform library
+* Kotlin Multiplatform library (with support for jvm, Linux, Windows
```
Comment (Scogun) [inline README.md:7]: Let's write support targets like a sublist:
````
Support:
  * JVM
  * Linux
  * ...
```

File: README.md (hunk 1 above)
Comment (stephanepechard) [reply README.md]: Done in 0099a2bc5cec40bed599ed0ce1414a93c7a27118

File: README.md (hunk 2 above)
Comment (stephanepechard) [reply README.md]: Done in 0099a2bc5cec40bed599ed0ce1414a93c7a27118

**Comment by Scogun** [comment]
Build agent in actions also should be change on `MacOS` one. Without it Mac artefacts won't be built:
```Some Kotlin/Native targets cannot be built on this linux_x64 machine and are disabled:
    * In project ':':
        * targets 'macosX64', 'iosArm64', 'iosX64', 'iosSimulatorArm64' (can be built with one of the hosts: macos_x64, macos_arm64)```

**Comment by Scogun** [comment]
> Build agent in actions also should be change on `MacOS` one. Without it Mac artefacts won't be built:
> 
> ```
>     * In project ':':
>         * targets 'macosX64', 'iosArm64', 'iosX64', 'iosSimulatorArm64' (can be built with one of the hosts: macos_x64, macos_arm64)```
> ```

Though, I don't really know do we need it. I didn't work with iOS so far. Where are Swift packages store?

**Comment by stephanepechard** [comment]
I guess one can use [Swift Package manager,](https://www.swift.org/package-manager/) but I don't use it directly, as the lib is used as a maven dependency of a bigger KMP project.

**Comment by stephanepechard** [comment]
I have to check on how to write a specific action for iOS packages, I'm not used to that, sorry.

**Comment by Scogun** [comment]
> I have to check on how to write a specific action for iOS packages, I'm not used to that, sorry.
I will create an issue for it.

**Review by Scogun — APPROVED** [review]

**Comment by Scogun** [comment]
The new version is released.
@stephanepechard thank you for contributing!

**Comment by stephanepechard** [comment]
@Scogun Thanks a lot!

**Pull request merged**